packages/config/config/devices/0x015f/mh-p220_3.2-255.255

```diff
@@ -0,0 +1,280 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
```

Comment:
Which one is it? S220 or P220? The device IDs look like the S220, the file is called p220 though.
I'm going to assume you meant S220.

Reply:
My apologies, its the S220.